src/plugins/deps/geomap-lib.js

```diff
@@ -1193,7 +1193,8 @@ var componentName = "wb-geomap",
 											atts = {};
 											for ( name in layerAttributes ) {
 												if ( layerAttributes.hasOwnProperty( name ) ) {
-													atts[ layerAttributes[ name ] ] = $row.find( name ).text();
+													att = $row.find( name ).text() ? $row.find( name ).text() : $row.find( "[ name='" +  name + "']" ).text();
```

Comment:
I don't think we need to cache the result in the att variable and we are doing <code>$row.find( name ).text() more than once. I suggest the following approach instead to optimize a bit better:
```js
name, rowText;
...
rowText = $row.find( name ).text();
atts[ layerAttributes[ name ] ] = rowText ? rowText : $row.find( "[ name='" +  name + "']" ).text();
```